CONTRIBUTING.md

```diff
@@ -32,12 +32,12 @@ Please consult the issues and discussions section of the SBT repo for good first
 ## Development
 
 1. Make desired changes
-1. Make sure existing (and new) tests pass successfully by running `npm run test`
-1. Run `npm run build` to compile
-1. Go to the root of the project. Then, deploy the CDK stack using the following:
-   - For `core-app-plane`: `npx cdk deploy --app='./lib/core-app-plane/integ.default.js'`
+2. Make sure existing (and new) tests pass successfully by running `npm run test`
+3. Run `npm run build` to compile
+4. Go to the root of the project. Then, deploy the CDK stack using the following:
    - For `control-plane`: `CDK_PARAM_SYSTEM_ADMIN_EMAIL="test@example.com" npx cdk deploy --app='./lib/control-plane/integ.default.js'`
-1. Test out the new feature(s) and redeploy as needed
-1. Write tests for any changed code and commit
-1. Create a PR.
-
+   - Once the control plane has been deployed, obtain the ARN of the newly deployed event bus [here](https://us-east-1.console.aws.amazon.com/events/home?region=us-east-1#/eventbuses)
```

Comment:
This is a region-specific URL, which will be confusing for customers deploying outside of `us-east-1`.

Maybe we can ask them to review the outputs of the ControlPlane stack to find the event bus ARN?